gcc/rust/util/rust-attributes.cc

```diff
@@ -41,6 +41,7 @@ static const BuiltinAttrDefinition __definitions[]
      {"repr", CODE_GENERATION},
      {"path", EXPANSION},
      {"macro_use", NAME_RESOLUTION},
+     {"target_feature", CODE_GENERATION},
```

Comment:
Can you add a `FIXME` comment here pointing to #1475 ? Since this is just a stub and doesn't actually implement the `target_feature` attribute, we need to be careful.

Reply:
Done! I've also reworded  the error message to sound a little better